pod/perlop.pod

```diff
@@ -2893,12 +2893,17 @@ This will print...
 
 ...with no leading whitespace.
 
-The delimiter is used to determine the B<exact> whitespace to
-remove from the beginning of each line.  All lines B<must> have
-at least the same starting whitespace (except lines only
-containing a newline) or perl will croak.  Tabs and spaces can
-be mixed, but are matched exactly.  One tab will not be equal to
-8 spaces!
+The line containing the delimiter that marks the end of the here-doc
+determines the indentation template for the whole thing.  Compilation
```

Comment:
This sentence feels cumbersome to me. Maybe
`Compilation croaks if any line does not begin with the precise indentation of the terminating line.`